Authorization/Middleware/DefaultSecurityMiddlewareOptions.cs

```diff
@@ -36,7 +38,18 @@ private Json CreateDefaultUnauthenticatedResponse(string deniedUri)
 
         private Response CreateDefaultUnauthorizedResponse()
         {
-            return new Response() { StatusCode = (ushort)System.Net.HttpStatusCode.NotFound };
+            var json = new Json
+            {
+                ["Html"] = _authenticationUriProvider.UnauthorizedViewUri
+            };
+
+            var response = new Response()
+            {
+                Resource = json,
+                StatusCode = (int) HttpStatusCode.NotFound
```

Comment:
This is sending a weird combination of status code and status text in the status line of the HTTP response. 

I was expecting `404 Not Found`, but actually sends `404 OK`:

![image](https://user-images.githubusercontent.com/566463/52792726-64f6ae00-306c-11e9-91f7-4c7b3c82a0af.png)

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/17431807/52796267-8e233880-3084-11e9-8a97-81cc03939184.png)